discover-rules/src/main/scala/org/xalgorithms/discover_rules/Job.scala

```diff
@@ -56,12 +56,13 @@ object Job {
     }).transform({rdd =>
       rdd.filter({tuple =>
         val filters = tuple._2.get[Option[UDTValue]]("filters")
-
+        // Make sure filters is defined
         filters.isDefined
       }).filter({tuple =>
         val filters = tuple._2.getUDTValue("filters")
         val envelope = filters.get[Option[Set[UDTValue]]]("envelope")
 
+        // Make sure envelope filters are defined
```

Comment:
As above... If the filters are undefined, will this filter eliminate the document?